assets/get-started-sidebar.js

```diff
@@ -1,26 +1,33 @@
-// Create the sidebar menus for each OS
+// Create the sidebar menus for each OS and Cloud Partner
 
 $([".macos", ".linux", ".windows"]).each(function(index, osClass) {
-  buildSidebarMenu(osClass);
+  buildSidebarMenu(osClass, "#get-started-locally-sidebar-list");
 });
 
-// On page load initially show the Mac OS menu
-
-showSidebar("macos");
-
-$("#macos").on("click", function() {
-  showSidebar("macos");
+$([".alibaba", ".aws", ".microsoft-azure", ".google-cloud"]).each(function(index, cloudPartner) {
+  buildSidebarMenu(cloudPartner, "#get-started-cloud-sidebar-list");
 });
 
-$("#linux").on("click", function() {
-  showSidebar("linux");
+// On start locally page load initially show the Mac OS menu
+showSidebar("macos", ".get-started-locally-sidebar li");
+
+$(["macos", "linux", "windows"]).each(function(index, osClass) {
+  $("#" + osClass).click(function() {
+    showSidebar(osClass, ".get-started-locally-sidebar li");
+  })
 });
 
-$("#windows").on("click", function() {
-  showSidebar("windows");
+// Show cloud partner side nav on click or hide side nav if already open 
+$(["alibaba", "aws", "microsoft-azure", "google-cloud"]).each(function(index, sidebarClass) {
+  $("#" + sidebarClass).click(function() {
+    showSidebar(sidebarClass, ".get-started-cloud-sidebar li");
+    if ($("#" + sidebarClass).parent().hasClass('open')) {
+      $(".get-started-cloud-sidebar li").hide();
+    }
+  })
 });
 
-function buildSidebarMenu(osClass) {
+function buildSidebarMenu(osClass, menuItem) {
```

Comment:
Instead of using `osClass` as the argument name, can you make it more general since both the local and cloud pages use this function?